src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -287,13 +286,13 @@
                                      (swap! state assoc :error (:message (get-parsed-response)))))}))
                     (swap! state assoc :error status-text)))}))
    :load-new-method-template
-   (fn [{:keys [state refs]} new-snapshot-id]
+   (fn [{:keys [state refs props]} new-snapshot-id]
      (let [[method-namespace method-name] (map (fn [key]
                                                  (get-in (:loaded-config @state)
                                                          [:methodConfiguration :methodRepoMethod key]))
                                                [:methodNamespace :methodName])
            config-namespace+name (select-keys (get-in @state [:loaded-config :methodConfiguration])
-                                              [:namespace :name])
+                                              [:namespace :name :rootEntityType])
            method-ref {:methodNamespace method-namespace
```

Comment:
This fixes the following bug:
When you edit a method config and then hit "Cancel", the root entity type changes from "participant" or "sample" or whatever to the string "rootEntityType". This is because the create-template endpoint initially sets it to that.